### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by aradradev**
In this pull request, I implemented the following features:

- Add any missing JSX elements to the project (without adding extra functionality)
- The final design should match zeplin's design (this includes all elements + the same styling)

Please review this pull request and any feedback on how I could enhance it is truly appreciated.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,63 +1,291 @@
...
+
+nav svg {
+  width: 40px;
+  color: var(--clr-white);
+}
+
+.navLinks {
+  display: flex;
+  align-items: center;
+  padding: 0.9rem;
```
Comment (Strangeal) [inline src/index.css]: - [ ] Great job on adding style to your project as mentioned in the project requirements! 👏 However, your styling doesn't match the original design, and due to that it doesn't adhere to some parts of the project requirements which state that 👇👇
Screenshot 2023-07-20 at 15 38 20


<hr />

`Note:` **Your styles should match exactly that of the `zeplin` design. Including `font-(color, size, weight)`, `background`, `buttons`, `input`, `spacing`, `layout`, etc. You're not to personalize any part of the design.** 


<table>
<tr>
<th align=“middle”>Replin design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
Screenshot 2023-07-20 at 15 47 28
</td>
<td>
Screenshot 2023-07-20 at 15 47 48
</td>
</tr>
</table>

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @aradradev, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well-structured project ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 2]
```diff
@@ -62,30 +64,17 @@ h1,
...
+  color: #121212;
 }
 
 p {
-  margin-bottom: 1.25rem;
+  color: #e8e8e8;
+  font-weight: bold;
 }
 
 @media screen and (min-width: 800px) {
```
Comment (algerina) [inline src/index.css]: - [ ] Well done working on the style however you still have some details to work on:
- Kindly use the same font and color as the Zeplin design .

Your design:

Screenshot 2023-07-20 at 7 28 34 PM

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @aradradev ,

Great job so far! 

May I highlight the following: 
- [x] You have a descriptive PR and commit history👏  ✔️ 
- [x] Your design look almost like the Zeplin design✔️ 




## CHANGES REQUIRED:muscle::clap:
Check the comments under the review.


 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @aradradev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 Great job making changes as requested by previous reviewers.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**